packages/beachball/src/__e2e__/monorepo/getScopedPackages.test.ts

```diff
@@ -0,0 +1,22 @@
+import { getScopedPackages } from '../../monorepo/getScopedPackages';
+import { BeachballOptions } from '../../types/BeachballOptions';
+import { MonoRepoFactory } from '../../fixtures/monorepo';
+
+describe('getScopedPackages', () => {
+  it('can scope packages', async () => {
+    const repoFactory = new MonoRepoFactory();
+    await repoFactory.create();
+    const repo = await repoFactory.cloneRepository();
+
+    const scopedPackages = getScopedPackages({
+      path: repo.rootPath,
+      scope: ['packages/grouped/*'],
+    } as BeachballOptions);
```

Comment:
we should avoid casting here and other places too. we can change to `getScopedPackages(options: Partial<BeachballOptions>`